modules/ffmpeg/producer/ffmpeg_producer.cpp

```diff
@@ -313,12 +316,12 @@ struct ffmpeg_producer : public core::frame_producer_base
 			                   // therefore no seeking should be necessary for the first frame.
 		{
 			input_.seek(file_position > 1 ? file_position - 2: file_position).get();
-            boost::this_thread::sleep_for(boost::chrono::milliseconds(40));
+			std::this_thread::sleep_for(std::chrono::milliseconds(40));
```

Comment:
Why have you changed this? This removes support for `boost::thread::interrupt()`.